livedev/livedev

```diff
@@ -131,6 +136,15 @@ class Remote(object):
             for action in actions:
                 action.run(self)
 
+    def create_ssh_id_if_not_exists(self):
+        cmd = 'ssh-keygen -t rsa -b 2048 -P "" -f ~/.ssh/id_rsa <<< n'
+        subprocess.run(cmd, shell=True)
+
+    def copy_ssh_id(self):
+        self.create_ssh_id_if_not_exists()
+        cmd = 'ssh-copy-id -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ~/.ssh/id_rsa.pub -f '+self.host
+        subprocess.run(cmd, shell=True)
```

Comment:
I prefer if the tool gracefully exits if there is no key rather than trying to create one.
It is not the place for this tool to create a ssh key in behalf of the user, especially with the risk of overriding an existing one.
Also the defaults that you set for the key are extremely low.
I for one do not believe in `rsa` anymore especially not below `8192` bits.